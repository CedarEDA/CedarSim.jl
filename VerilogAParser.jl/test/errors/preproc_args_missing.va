`define MPRco(nam,def,uni,lwr,upr,des) (*units=uni, desc=des*) parameter real nam=def from[lwr:upr) ;
module preproc_args_missing(p, n)
    `MPRco( DLCIG         , LINT           ,""                                      , "Source/drain overlap length for Igs")
endmodule