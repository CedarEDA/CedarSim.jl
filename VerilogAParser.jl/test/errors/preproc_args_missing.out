[31m[1mâ•­ ERROR [VAS47]:[22m[39m[31m Expected semicolon[39m[31m.[39m
[31mâ”‚[39m[90m  â•­â”€â”€â”€â”€â”€ preproc_args_missing.va:2[39m
[31mâ”‚[90m2[0m [90mâ”‚[0m module preproc_args_missing(p, n)[91mâ—Œ 
[31mâ”‚[0m  [90mâ”Š[0m                                  [32m;[0m
[31mâ”‚[39m[90m  â”‚[39m
[31mâ”‚[39m[90m  â•°â”€â”€â”€â”€â”€ [39m Missing semicolon.
[31mâ”‚[39m
[31mâ”‚[0m [34mhelp:[0m The parser considered the declaration complete at the indicated location and thus expected a semicolon. If you expected the subsequent code to be part of this declaration, check for missing syntax (e.g. binary operators).
[31mâ•¯[39m